index.js

```diff
@@ -76,7 +76,7 @@ class ExecutorQueue extends Executor {
         this.fuseBox.addFuse(this.redisBreaker);
 
         const retryOptions = {
-            plugins: ['retry'],
+            plugins: ['Retry'],
             pluginOptions: {
                 retry: {
```

Comment:
need to fix this key as https://github.com/taskrabbit/node-resque/blob/master/examples/retry.js#L25-L27

Reply:
fixed :)